src/Context/LikeContext.js

```diff
@@ -0,0 +1,66 @@
+import axios from "axios";
+import { createContext, useContext, useState, useEffect} from "react";
+import { toast } from "react-toastify";
+
+
+const LikeContext = createContext();
+const useLikes = () => useContext(LikeContext)
+
+const LikeContextProvider = ({children}) =>{
+    const [likes, setLikes] = useState([])
+    useEffect(()=>{
+        getLike()
+    },[])
```

Comment:
suggestion: you should include getLike in the dependency array. You can react memoization to do that.